notebooks/amplicon_pre_prep_file_generator.ipynb

```diff
@@ -21,6 +21,14 @@
    "source": [
     "# Knight Lab Amplicon Mapping File (Pre-Preparation File) Generator\n",
     "\n",
+    "<font color='red'><B>A VERY IMPORTANT Note on Plate Compression and Positions:</B></font>\n",
+    "\n",
+    "This notebook works SPECIFICALLY with the STANDARD EpMotion compression format ONLY.\n",
+    "PRIMERS are tied to our standard SPECIFIC PLATE POSITIONS outlined in the code, and CANNOT be changed.\n",
+    "\n",
+    "If you are not compressing in the standard compression format (position 1, position 2, position 3, position 4) that uses the plate map template below,\n",
+    "DO NOT USE THIS NOTEBOOK TO GENERATE THE MAPPING FILE.\n",
```

Comment:
mapping file?? is this the pre-preparation file?

Reply:
Good catch, Antonio. I am still using old language.
Charlie - Can you please change "mapping file" to "pre-preparation file" in lines 30? Thank you.

Reply:
@mmbryant23 line changed, but just I noticed mapping file is used in several other places in the notebook as well - sometimes pre-prep is in parentheses but sometimes not. Should we go ahead and remove mapping-file entirely?